src/main.scss

```
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+  font-family: Arial;
+  font-size: 100px;
+  color: white;
```

```suggestion
  color: #fff;
```
код предпочтительнее чем строка